backend/api/build_kfp_server_api_python_package.sh

```diff
@@ -34,28 +34,20 @@ if [ -z "$VERSION" ]; then
     exit 1
 fi
 
-codegen_file=/tmp/swagger-codegen-cli.jar
-# Browse all versions in: https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.7/
-codegen_uri=https://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.4.7/swagger-codegen-cli-2.4.7.jar
+codegen_file=/tmp/openapi-generator-cli.jar
+# Browse all versions in: https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/
+codegen_uri="https://repo1.maven.org/maven2/org/openapitools/openapi-generator-cli/4.3.1/openapi-generator-cli-4.3.1.jar"
 if ! [ -f "$codegen_file" ]; then
-    wget --no-verbose "$codegen_uri" -O "$codegen_file"
+    curl -L "$codegen_uri" -o "$codegen_file"
 fi
 
 pushd "$(dirname "$0")"
 
-DIR=$(mktemp -d)
-
-swagger_file=$(mktemp)
-
-echo "Merging all Swagger API definitions to $swagger_file."
-jq -s '
-    reduce .[] as $item ({}; . * $item) |
-    .info.title = "KF Pipelines API" |
-    .info.description = "Generated python client for the KF Pipelines server API"
-' ./swagger/{run,job,pipeline,experiment,pipeline.upload}.swagger.json > "$swagger_file"
+DIR="$(pwd)/python_http_client"
```

Comment:
I wonder should we delete the directory?
Otherwise, when renaming or deleted objects, old files will still remain.

Reply:
Right, that's a good point!

Reply:
Fixed